symplyphysics/laws/relativistic/vector/relativistic_velocity_orthogonal_to_movement.py

```diff
@@ -0,0 +1,73 @@
+from sympy import sqrt
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    dot_vectors,
+    scale_vector,
+    QuantityVector,
+)
+from symplyphysics.core.vectors.arithmetics import reject_cartesian_vector
+
+# Description
+## Consider two inertial reference frames: one fixed (lab frame) and one tied to the moving object (proper frame).
+## The proper frame is moving with some velocity `v` relative to the lab frame. Then, according to the theory of special
+## relativity, the velocity of the object relative to lab frame is not equal to the sum of its velocity in the proper
+## frame and the velocity of the proper frame relative to the lab frame.
+
+# Law: u_orthogonal = u_orthogonal' / (gamma * (1 - dot(v, u_parallel') / c**2))
+## u_orthogonal - velocity vector relative to lab frame orthogonal to `v`
+## u_orthogonal' - velocity vector relative to proper frame orthogonal to `v`
+## u_parallel' - velocity vector relative to proper frame parallel to `v`
+## v - velocity vector of proper frame relative to lab frame
+## gamma = 1 / sqrt(1 - dot(v, v) / c**2) - Lorentz factor
+## dot(a, b) - dot product between vectors `a` and `b`
+
+# Notes
+## - One can get the same expression for `u_orthogonal'` in terms of `u_orthogonal` by replacing `v` with `-v`. This is
+##   essentially the inverse Lorentz transformation from lab frame to proper frame that uses the fact that the lab frame
+##   can be viewed as moving with velocity vector `-v` relative to the proper frame.
+
+# Conditions
+## - Works in special relativity
+
+
+def orthogonal_velocity_component_in_lab_frame_law(
+    velocity_in_proper_frame_: Vector,
+    proper_frame_velocity_: Vector,
+) -> Vector:
+    orthogonal_velocity_component_in_proper_frame_ = reject_cartesian_vector(velocity_in_proper_frame_, proper_frame_velocity_)
+
+    lorentz_factor_ = 1 / sqrt(1 - dot_vectors(proper_frame_velocity_, proper_frame_velocity_) / speed_of_light**2)
+
+    scale_factor_ = lorentz_factor_ * (1 + dot_vectors(proper_frame_velocity_, velocity_in_proper_frame_) / speed_of_light**2)
```

Comment:
In description there is parallel component inside the dot product: `dot_vectors(proper_frame_velocity_, velocity_in_proper_frame_projection)`

Reply:
They yield to the same result and I wanted this law to be similar to the parallel counterpart.

Reply:
You should either project it or fix description. If you'll project vector, add small comment, that the result is the same, as when using unprojected vector.

Reply:
Sure

Reply:
I am still not happy with this approach. In the description above there is parallel vector. As I said before, you should either update law description, or explicitly project vector.

Reply:
I went with fixing the description